src/activities/guesscount/Guesscount.qml

```diff
@@ -0,0 +1,462 @@
+                model:3
+                DropArea {
+                    id: dragTarget
+                    width: 100
+                    height: 100
+                    property bool present
+                    property var mousearea: dragTarget.children[1]
+                    property var calculated_value
+
+
+
+                    onDropped: {
+                        dragTarget.present=Qt.binding(function() { return true })
+                        dragTarget.keys="not_allowed"
+                        if(index==1)
+                        {
+                            var p=dragTarget.children[1]
+                            console.log(p.obj.operation)
+                        }
+                        else
+                        {
+                            var p=dragTarget.children[1]
+                            console.log(p.obj.datavalue)
+                        }
+                        var m=row4.children[0]
+
+                        if(row4.children[0].present && row4.children[1].present && row4.children[2].present)
+                        {
+                            switch (row4.children[1].children[1].obj.operation) {
+                            case "+":
+                                console.log(row4.children[0].children[1].obj.datavalue+row4.children[2].children[1].obj.datavalue);
+                                dragTarget.calculated_value=row4.children[0].children[1].obj.datavalue+row4.children[2].children[1].obj.datavalue
+                                break;
+                            case "-":
+                                console.log(row4.children[0].children[1].obj.datavalue-row4.children[2].children[1].obj.datavalue);
+                                dragTarget.calculated_value=row4.children[0].children[1].obj.datavalue-row4.children[2].children[1].obj.datavalue
+                                break;
+                            case "/":
+                                console.log(row4.children[0].children[1].obj.datavalue/row4.children[2].children[1].obj.datavalue);
+                                dragTarget.calculated_value=row4.children[0].children[1].obj.datavalue/row4.children[2].children[1].obj.datavalue
+                                break;
+                            default:
+                                console.log(row4.children[0].children[1].obj.datavalue*row4.children[2].children[1].obj.datavalue);
+                                dragTarget.calculated_value=row4.children[0].children[1].obj.datavalue*row4.children[2].children[1].obj.datavalue
+
+                            }
+                            row4.children[5].children[0].text=Number(dragTarget.calculated_value).toString()
+                            if(dragTarget.calculated_value==items.guesscount)
+                            {
+
+                                row4.children[5].children[0].text=Number(dragTarget.calculated_value).toString()
+                                if(items.question_no==3){
```

Comment:
can you uniform the coding style, after the if(), there is a space and the accolade:
if(test) { 
   ...
}